include/range/v3/view/split.hpp

```diff
@@ -202,6 +209,7 @@ namespace ranges
                 using FunctionConcept = meta::and_<
                     ForwardRange<Rng>,
                     Invocable<Fun&, iterator_t<Rng>, sentinel_t<Rng>>,
+                    Invocable<Fun&, iterator_t<Rng>, iterator_t<Rng>>,
```

Comment:
This additional constraint is really "breaking" since the implementation was already relying on it implicitly (See line 114).